llm_bench/python/benchmark.py

```diff
@@ -725,6 +725,9 @@ def get_argprser():
 def main():
     logging_kwargs = {"encoding": "utf-8"} if sys.version_info[1] > 8 else {}
     log.basicConfig(format='[ %(levelname)s ] %(message)s', level=os.environ.get("LOGLEVEL", log.INFO), stream=sys.stdout, **logging_kwargs)
+    env_omp = os.getenv('OMP_WAIT_POLICY')
+    if env_omp is None or env_omp != 'PASSIVE':
+        log.warning('The environment OMP_WAIT_POLICY is not set to PASSIVE')
```

Comment:
```suggestion
        log.warning('It is recommended to set the environment variable OMP_WAIT_POLICY to PASSIVE, so that OpenVINO inference can use all CPU resources without waiting.')
```